src/features/cookies/helpers/removeGenerateCookieKey.test.ts

```diff
@@ -0,0 +1,38 @@
+import { removeGenerateCookieKey } from 'features/cookies/helpers/removeGenerateCookieKey'
+import { storage, StorageKey } from 'libs/storage'
+import { renderHook, superFlushWithAct } from 'tests/utils'
+
+const key = 'traffic_medium'
+const keyPrefix = 'algoliasearch-client-js' as StorageKey
+const keyWithPrefix = keyPrefix.concat('-', '1234') as StorageKey // algoliasearch-client-js-1234
+const keyWithPartialPrefix = keyPrefix.slice(0, keyPrefix.length / 2) as StorageKey // algoliasear
+const someValue = 'some value'
+
+describe('removeGenerateCookieKey', () => {
+  it('should clear optional key', async () => {
+    storage.saveObject(key, someValue)
+
+    renderHook(() => removeGenerateCookieKey(key))
+
+    await superFlushWithAct()
+    expect(await storage.getAllKeys()).toEqual([])
+  })
+
+  it('should clear optional key with prefix', async () => {
+    storage.saveObject(keyWithPrefix, someValue)
```

Comment:
tel que je l'ai compris, dans le storage il devrait il y avoir la clé entière

```suggestion
    storage.saveObject(key, someValue)
```

Reply:
La constante key c'est une key qui n'a pas de préfix et qu'on retrouve dans StorageKey. C'est utilisé pour le premier test. 

Dans ce cas la, c'est bien la clé avec le préfix qu'on save, et on regarde si l'appel de la fonction avec le préfix 
 `removeGeneratedStorageKey(keyPrefix)` supprimer bien la key du local storage